aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1658,4 +1658,86 @@ Tensor& diag_cpu_out(Tensor &result, const Tensor& self, int64_t dimension) {
   return result;
 }
 
+Tensor movedim(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
+  TORCH_CHECK(src.size() == dst.size(), "movedim: Invalid source or destination dims: src (",
+              src, " dims ) should contain the same number of dims as dst (", dst, " dims)");
```

Comment:
To be consistent, we should also change "src" -> "source" and "dst -> destination" in the error messages.

Reply:
Right! Missed that Thank You!

Reply:
Done! Thanks